pkg/disk/partition_table.go

```diff
@@ -143,9 +143,9 @@ func NewPartitionTable(basePT *PartitionTable, mountpoints []blueprint.Filesyste
 
 	// If no separate requiredSizes are given then we use our defaults
 	if requiredSizes == nil {
-		requiredSizes = map[string]uint64{
-			"/":    1073741824,
-			"/usr": 2147483648,
+		requiredSizes = map[string]datasizes.Size{
+			"/":    datasizes.Size(1073741824),
+			"/usr": datasizes.Size(2147483648),
```

Comment:
Nitpick: could we use the size constants here instead of the crazy number? :innocent:

Reply:
Sure!